packages/core/package.json

```diff
@@ -107,5 +107,8 @@
   },
   "nohoist": [
     "requirejs"
-  ]
+  ],
+  "bin": {
+    "copy-thebe-assets": "./bin/copy-thebe-assets.cjs"
```

Comment:
This is great. I am using this over in another myst-theme as well, and it will be nice to not have to copy this script all over the place!